server/WAPI.rb

```diff
@@ -252,6 +272,31 @@ def get_request(request)
         wrapped_response = do_request(request)
       end
     end
+
+    # If it didn't work just return that information.
+    logger.debug "#{self.class.to_s}:#{__method__}:#{__LINE__}: wrapped_response: meta_status: #{wrapped_response.meta_status}"
+    if wrapped_response.meta_status != WAPI::SUCCESS
+      return wrapped_response
+    end
+
+    # Ran a query successfully.  See if got partial data and need to keep going.
```

Comment:
The comment is describing the next processing to be done and it doesn't necessarily only apply to only things before the next comment.  That isn't clear enough.